README.md

```diff
@@ -1,197 +1,195 @@
 
-## Quickstart
-
-1. Clone the repo locally:
 
-```bash
-  git clone -b mocknet --depth 1 https://github.com/blockstack/stacks-local-dev ./stacks-local-dev && cd ./stacks-local-dev
-```
-
-2. Create/Copy `.env` file
-*Use a symlink as an alternative to copying: `ln -s sample.env .env`*
-```bash
-  cp sample.env .env
-```
+### Install/Update docker-compose
+*Note: `docker-compose` executable is required, even though recent versions of Docker contain `compose` natively*
 
-3. Start the Services:
+First, check if you have `docker-compose` installed locally:
 ```bash
-docker-compose up -d
+$ docker-compose --version
+docker-compose version 1.27.4, build 40524192
 ```
-*NOTE*: We are now importing V1 BNS data. What this means is that initially, there will be a longer startup time while the data is downloaded and extracted, then loaded via the API container into postgres. Once this initial load is complete, subsequent restarts will be much faster. Additionally, all data is persistent here - postgres and the stacks-blockchain, so bringing a node up to the tip height should be much faster. 
 
-4. Stop the Services:
+If the command is not found, or the version is < `1.27.4`, run the following to install the latest to `/usr/local/bin/docker-compose`:
 
 ```bash
-docker-compose down
+VERSION=$(curl --silent https://api.github.com/repos/docker/compose/releases/latest | jq .name -r)
+DESTINATION=/usr/local/bin/docker-compose
+sudo curl -L https://github.com/docker/compose/releases/download/${VERSION}/docker-compose-$(uname -s)-$(uname -m) -o $DESTINATION
+sudo chmod 755 $DESTINATION
 ```
+[Install Docker-compose](https://docs.docker.com/compose/install/)
+[Docker Desktop for Mac](https://docs.docker.com/docker-for-mac/install/)
+[Docker Desktop for Windows](https://docs.docker.com/docker-for-windows/install/)
+[Docker Engine for Linux](https://docs.docker.com/engine/install/#server)
 
-## Bootstrap Container
-
-The first container to start will always be the `mocknet_bootstrap` container. The sole purpose of this container is to run a [script](https://github.com/blockstack/stacks-local-dev/blob/mocknet/setup.sh) to replace the variables in the `.template` files with the values from `.env`
-
-## API Container
-
-The API Container will run a [script](https://github.com/blockstack/stacks-local-dev/blob/mocknet/setup-bns.sh) before starting it's server. The sole purpose of this is to download (or verify the files exist) V1 BNS data. Once the download/extraction/verification has finished, the `stacks-blockchain-api` server will start up
 
-## Env Vars
 
-All variables used in the [`sample.env`](https://github.com/blockstack/stacks-local-dev/blob/mocknet/sample.env) file can be modified, but generally most of them should be left as-is.
 
-## Local Data Dirs
+### Env Vars
+All variables used in the [`sample.env`](https://github.com/blockstack/stacks-local-dev/blob/main/sample.env) file can be modified, but generally most of them should be left as-is.
```

Comment:
We should be able to make those kinds of links relative right? In case we move/rename the repo.

Reply:
In theory, yes - i'll try it out and see if it works as expected